setup.py

```diff
@@ -110,6 +110,7 @@ def get_git_sha():
     extras_require={
         "bigquery": ["pybigquery>=0.4.10", "pandas_gbq>=0.10.0"],
         "cors": ["flask-cors>=2.0.0"],
+        "flask-testing": ["Flask-Testing==0.7.1"],
```

Comment:
Nit. Can we lowercase this like the other packages. Package names are case insensitive.